Python_Scripts/C.py

```diff
@@ -56,9 +56,11 @@
 # Base folder for all models
 DIR_MODELS = os.path.join(DIR_BASE, 'Models')
 # Folder for sedml and sbml model definitions
-DIR_MODELS_SEDML = os.path.join(DIR_MODELS, 'sedml')
+DIR_MODELS_SEDML = os.path.join(DIR_MODELS, 'sedml')  # TODO remove
 # Folder for a copy of the biomodels models
 DIR_MODELS_BIOMODELS = os.path.join(DIR_MODELS, 'biomodels')  # TODO remove
+# Folder with the final benchmark models (regrouped and adapted sbml files)
+DIR_MODELS_FINAL = os.path.join(DIR_MODELS, 'benchmark_collection')
```

Comment:
I guess this will then replace DIR_MODELS_AMICI_FINAL?

Reply:
Well, we have an additional step in between now...
we used to have `SED-ML-models  -->  AMICI models`
now we have
`SED-ML-models  -->  regrouped model` and `regrouped models  -->  AMICI models`